components/brave_wallet/common/pref_names.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_COMMON_PREF_NAMES_H_
+#define BRAVE_COMPONENTS_BRAVE_WALLET_COMMON_PREF_NAMES_H_
+
+namespace brave_wallet {
+namespace prefs {
+
+// Used to enable/disable Brave Wallet via a policy.
+inline constexpr char kDisabledByPolicy[] = "brave.wallet.disabled_by_policy";
```

Comment:
why `inline`?

Reply:
Upstream seems to be doing it this way (https://source.chromium.org/chromium/chromium/src/+/main:components/sync/base/pref_names.h).